tests/stdlib/tparsesql.nim

```diff
@@ -0,0 +1,521 @@
+import unittest
+
+import sequtils
+import strutils
+import parsesql
+
+proc fold(str: string): string =
```

Comment:
What's all this logic doing here? Remove it, tests should be simple.

Reply:
This makes it so indented sample code matches.